test/e2e/policies.test-suite/topology-aware/n4c16/test10-additional-reserved-namespaces/code.var.sh

```diff
@@ -28,3 +28,22 @@ verify 'cpus["pod0c0"] == {"cpu10", "cpu11"}'
 verify 'cpus["pod1c0"] == {"cpu08", "cpu09"}'
 
 cleanup-test-pods
+
+# Test that
+# - conatainers that are annotated to opt out from reserved pools are pinned elsewhere
+
+(kubectl create namespace foobar) || true
+
+cleanup-foobar-namespace() {
+    (kubectl delete pods -n foobar --all) || true
+}
+cleanup-foobar-namespace
+
+CONTCOUNT=1 namespace=foobar create besteffort
+ANN0='prefer-reserved-cpus.cri-resource-manager.intel.com/pod: "false"'
+CONTCOUNT=1 namespace=foobar create besteffort
```

Comment:
Suggesting one more pod and check for catching regression if this logic is changed later on:
```
+ANN0='prefer-reserved-cpus.cri-resource-manager.intel.com/pod: "true"'
+CONTCOUNT=1 namespace=foobar create besteffort
 report allowed
 verify 'cpus["pod2c0"] == {"cpu10", "cpu11"}'
 verify 'cpus["pod3c0"] == {"cpu08", "cpu09"}'
+verify 'cpus["pod4c0"] == {"cpu10", "cpu11"}'
```
(That is, both namespace and annotation point to preferring reserved... The reason why I suggest this is the `switch... case checkReservedPoolNamespaces(namespace) && !explicitReservation` where obviously `!explicitReservation` is `false` when the annotation is set.)

Reply:
Otherwise this looks good.

Note to self: as this is not policy-specific annotation, I think we should have this in balloons and podpools policies, too.